Mjml.Net.Benchmarking/TemplateBenchmarks.cs

```diff
@@ -5,84 +5,60 @@ namespace Mjml.Net.Benchmarking
     public class TemplateBenchmarks
     {
         private readonly MjmlRenderer MjmlRenderer;
-        public string TemplateAmario { get; set; }
-        public string TemplateAustin { get; set; }
-        public string TemplateSphero { get; set; }
-        public string TemplateManyHeroes { get; set; }
+     
+        [ParamsSource(nameof(MjmlTemplates))]
+        public string MjmlTemplateFileName { get; set; }
 
-        public TemplateBenchmarks()
-        {
-            MjmlRenderer = new();
+        public IEnumerable<string> MjmlTemplates => new[] {
```

Comment:
I would just loop over the directory => Directory.GetFiles("*.mjml").Select(x => x.Name);  or so

Reply:
The point is that we can easily add more templates and do not have to change the code for that.